tasks/install.yaml

```
@@ -0,0 +1,34 @@
+- block:
+
+  - name: Add MariaDB repository key
+    apt_key:
+      url: "{{ mariadb.repository.apt.key_url }}"
+
+  - name: Add MariaDB repository
+    apt_repository:
+      repo: "{{ mariadb.repository.apt.repository }}"
+      filename: /etc/apt/sources.list.d/mariadb.list
+
+  when: ansible_distribution == 'Ubuntu'
+
+- name: update apt cache
+  apt:
+    update_cache: yes
+  when: ansible:distribution == 'Ubuntu'
+
+- name: Install MariaDB
+  when:
+    - ansible_distribution == 'Ubuntu'
+    - mariadb.galera.arbitrator
```

Not sure but should this be an `and not mariadb.galera.arbitrator`?
```suggestion
    - not mariadb.galera.arbitrator
```